src/Components/Atoms/FASwitch/index.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { memo } from "react";
+import { Switch, FormControlLabel } from "@mui/material";
```

Comment:
exclude label from switch component and create its component in FAFormControlLabel.

Reply:
okay